src/api/app/views/layouts/webui2/_footer.html.haml

```diff
@@ -4,46 +4,40 @@
       %strong
         = User.current
       %ul
-        %li= link_to "Your Profile", user_show_path(User.current)
+        %li= link_to 'Your Profile', user_show_path(User.current)
```

Comment:
What about adding parenthesis on all the `link_to` calls?

Reply:
Right now we use have link_to calls with and without parenthesis in OBS. And within this file all link_to call are without them.
So I don't see a point in changing for this one line, or even this single file.

Reply:
@bgeuken, fine for me but, we need to have it in mind for webui2.

Reply:
@DavidKang Can you elaborate? How is this connected to webui2 (and not OBS in general)?

Reply:
I've updated that file now, since I was rebasing the commits anyway. Though if you want this to be the standard we would still have to update all the other OBS views:wink:

Reply:
@bgeuken, the footer of webui2 is using `%a` instead of `link_to`. We also need to apply this changes for webui2 footer.

Reply:
@bgeuken, forget about what I said, I was thinking that was the footer of `webui/footer`